Delicious.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+  <html>
+      <head>
+      <title>Delicous Blog</title>
+      <link href="css/style.css" type="text/css" rel="stylesheet" />
+      <link href='http://fonts.googleapis.com/css?family=Yellowtail' rel='stylesheet' type='text/css'>
+    </head>
+   
+    <body>
+      <header>
+      		<img src = "img/logo.png" />
+      		<section>
+      			<h1>Delicious</h1>
+      			<h2>THE BEST FOOD BLOG ON THE WEB.</h2>
+      		</section>
+      		<nav>
+      			<a href = ""><img src = "img/fb-icon.png" /></a>
+          	<a href = ""><img src = "img/twit-icon.png" /></a>
+          	<a href = ""><img src = "img/gp-icon.png" /></a>
+          	<a href = ""><img src = "img/insta-icon.png" /></a>
+          	<a href = ""><img src = "img/flic-icon.png" /></a>
+          	<a href = ""><img src = "img/pint-icon.png" /></a>
+          	<a href = "" id = "rss"><img src = "img/rss-icon.png" /></a>
+         		<a href = ""><img src = "img/mail-icon.png" /></a>
+        	</nav>
+      </header>
+      <hr />
```

Comment:
do not use hr tags. They are the perfect example of using HTML for presentation, which is a no-no. Use CSS to do anything presentational.